### ISSUE TITLE
Where to find compile commands and result?

### ISSUE THREAD

**Issue by GitMensch** [mridoni/gix#113]
I've expected to find that in the Output pane but find no place where it actually is available.

**Comment by mridoni** [comment]
No, it's just not there :-) I was already thinking of dividing the output pane into sub-panes, and separating the preprocessor and compiler output from the diagnostic (or other) output. This will also allow the (easier) introduction of a separate "Errors and Warnings pane", that if double-clicked will lead to the source of the error/warning.
